tests/apollo/test_skvbc_view_change.py

```diff
@@ -59,6 +60,8 @@ async def test_request_block_not_written_primary_down(self, bft_network):
         4) Verify the BFT network eventually transitions to the next view.
         5) Validate that there is no new block written.
         """
+        bft_network.do_key_exchange()
```

Comment:
`await  bft_network.do_key_exchange()`, otherwise it's not executed

Reply:
thanks